Azure-Samples/java-docs-spring-hello-world
# Hello World sample code for the Java documentation for Azure App Service

## Purpose

Adding the Hello World code to the repo. This will be used by the documentation to set up a Java11 Java SE app using the Azure portal.

## Does this introduce a breaking change?

[ ] Yes
[X] No


## Pull Request Type
What kind of change does this Pull Request introduce?

[ ] Bugfix
[X] Feature - Hello World!
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:

## How to Test

* Fork the repo
* Attach to a Java 11, Java SE Azure App Service
* Watch it deploy
* Confirm you see "Hello World" in the Azure App Service


## What to Check

Most files are boilerplate. However, I made changes to the following files:

- README.md - more descriptive, more aligned with the other sample repos
- pom.xml - updated name and description
- DemoApplication.java - added a REST endpoint to display the Hello message


## Other Information
